### ISSUE TITLE
Generated html in p5.js mode doesn't have doctype declaration.

### ISSUE THREAD

**Issue by mashimolab** [benfry/processing4#465]
Hi, I realized that generated html in p5.js mode doesn't have doctype declaration.
If we don't have the declaration in html, color style of table tags does not work. (It is automatically set "color: -internal-quirk-inherit;").  So, if it is not the specification, i hope that the html is generated with html5 declaration as default.

(I checked it in processing4.0b7)

**Comment by BeyondBinaryMinds** [comment]
Ok thanks so much!!On Sat, Apr 9, 2022 at 11:15 AM mashimolab ***@***.***> wrote:
Hi, I realized that generated html in p5.js mode doesn't have doctype declaration.
If we don't have the declaration in html, color style of table tags does not work. (It is automatically set "color: -internal-quirk-inherit;").  So, if it is not the specification, i hope that the html is generated with html5 declaration as default.
(I checked it in processing4.0b7)

?Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>
